samwise.js

```diff
@@ -1,13 +1,12 @@
-const isPalindrome = word => word === word.split("").reverse().join("");
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?